intern.json

```diff
@@ -13,10 +13,6 @@
 		"./dist/dev/src/common/tests/unit/all.js"
 	],
 
```

Comment:
I believe more of this file can be removed but I am unsure which parts.

Reply:
I'm pretty sure everything with browserstack, saucelabs, selenium are not going to be needed for unit tests. Maybe try taking out most of the configuration here and see if the unit tests still run.